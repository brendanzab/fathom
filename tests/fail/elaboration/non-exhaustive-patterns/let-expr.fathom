//~ exit-code = 1

let 1 : U32 = 2;

{}
