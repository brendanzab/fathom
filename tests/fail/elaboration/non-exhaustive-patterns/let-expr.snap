stdout = ''
stderr = '''
error: non-exhaustive patterns in match expression
  ┌─ tests/fail/elaboration/non-exhaustive-patterns/let-expr.fathom:3:15
  │
3 │ let 1 : U32 = 2;
  │               ^ patterns not covered

'''
